OpenSim/Moco/MocoGoal/MocoContactImpulseTrackingGoal.cpp

```diff
@@ -179,76 +175,84 @@ void MocoContactTrackingGoal::initializeOnModelImpl(const Model& model) const {
         groupInfo.refExpressedInFrame = nullptr;
         if (extForce.get_force_expressed_in_body() != "ground") {
             const auto& forceExpressedInBody =
-                    extForce.get_force_expressed_in_body();
+                extForce.get_force_expressed_in_body();
             if (model.hasComponent<PhysicalFrame>(forceExpressedInBody)) {
                 groupInfo.refExpressedInFrame =
-                        &model.getComponent<PhysicalFrame>(
-                                forceExpressedInBody);
-            } else if (model.hasComponent<PhysicalFrame>(
-                               "./bodyset/" + forceExpressedInBody)) {
+                    &model.getComponent<PhysicalFrame>(
+                        forceExpressedInBody);
+            }
+            else if (model.hasComponent<PhysicalFrame>(
+                "./bodyset/" + forceExpressedInBody)) {
```

Comment:
This line should be indented once. (As a rule of thumb, if the if-statement expression needs multiple lines, you should indent the extra lines so they are not left-aligned with the code within the if-statement.)